### PR TITLE
Support configuration databaseId.

### PR DESCRIPTION
**Pull request by nieqiurong**
https://github.com/mybatis/spring-boot-starter/issues/911

### CODE REVIEW

File: mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisProperties.java [hunk 1]
```diff
@@ -409,6 +409,11 @@ public static class CoreConfiguration {
      */
     private Properties variables;
 
+    /**
+     * Specifies the databaseId
```
Comment (kazuki43zoo) [inline mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisProperties.java:413]: please JavaDoc end with '.' and I prefer description such as `"Specifies the database identify value for switching query to use."`.

**Review by kazuki43zoo — COMMENTED** [review]
@nieqiurong Thank you for your contribution!
I will accept your changes, but I added a small comment, so please check it.

File: mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisProperties.java (hunk 1 above)
Comment (nieqiurong) [reply mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisProperties.java]: > please JavaDoc end with '.' and I prefer description such as `"Specifies the database identify value for switching query to use."`.

Thank you for your suggestion, I have adjusted it

**Comment by kazuki43zoo** [comment]
@nieqiurong If you have a time, please modify tests of `MybatisPropertiesTest`.

* Add assertion on `testWithDefaultCoreConfiguration`
* Add a property of `databaseId` and assertion on `testWithCustomizeCoreConfiguration`
* Remove the `databaseId` from https://github.com/mybatis/spring-boot-starter/blob/master/mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisPropertiesTest.java#L249.

If you are busy, I will perform those.

**Comment by nieqiurong** [comment]
> @nieqiurong If you have a time, please modify tests of `MybatisPropertiesTest`.
> 
> * Add assertion on `testWithDefaultCoreConfiguration`
> * Add a property of `databaseId` and assertion on `testWithCustomizeCoreConfiguration`
> * Remove the `databaseId` from https://github.com/mybatis/spring-boot-starter/blob/master/mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisPropertiesTest.java#L249.
> 
> If you are busy, I will perform those.

Hi i'm done

**Pull request merged**